src/challenges.js

```diff
@@ -1,49 +1,167 @@
 // Desafio 1
-function compareTrue() {
+function compareTrue(firstValue, secondValue) {
   // seu código aqui
+  if (firstValue === true && secondValue === true) {
```

Comment:
@CIGR2021  você pode simplificar essa linha escrevendo assim: 
```suggestion
  if (firstValue && secondValue) {
```

entende o porquê?